app/screens/AddExpectingChildProfile.tsx

```diff
@@ -59,6 +60,9 @@ const styles = StyleSheet.create({
   headerRowView: {
     maxHeight: 50
   },
+  headetTitleText: {

```

Comment:
headerTitleText   ???

Reply:
@Akhror I will rename this headetTitleTextColor.

Reply:
Why headetTitleTextColor? It should be headerTitleTextColor

Reply:
okay @Akhror , I will rename to headerTitleTextColor.